ensime_shared/ensime.py

```diff
@@ -822,15 +826,11 @@ def detect_row_column_start():
 
 
 class EnsimeClientV1(ProtocolHandlerV1, EnsimeClient):
-    def __init__(self, *args, **kwargs):
```

Comment:
Ugh, so sorry, I was running the code on this branch but I wrote an en-dash in a docstring in the last commit and didn't realize that broke Python at the last minute 😩   Fixed now, this code should run.